refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/IsEmptyTest.java

```diff
@@ -0,0 +1,331 @@
+package tech.picnic.errorprone.refaster.matchers;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.CompilationTestHelper;
+import com.google.errorprone.bugpatterns.BugChecker;
+import org.junit.jupiter.api.Test;
+
+final class IsEmptyTest {
+  @Test
+  void matches() {
+    CompilationTestHelper.newInstance(MatcherTestChecker.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "import com.google.common.collect.ImmutableList;",
+            "import com.google.common.collect.ImmutableMap;",
+            "import com.google.common.collect.ImmutableSet;",
+            "import com.google.common.collect.ImmutableSetMultimap;",
+            "import java.util.ArrayList;",
+            "import java.util.Collections;",
+            "import java.util.HashMap;",
+            "import java.util.LinkedHashMap;",
+            "import java.util.LinkedList;",
+            "import java.util.List;",
+            "import java.util.Map;",
+            "import java.util.Set;",
+            "import java.util.Stack;",
+            "import java.util.Vector;",
+            "import java.util.stream.Stream;",
+            "import reactor.core.publisher.Flux;",
+            "import reactor.core.publisher.Mono;",
+            "import reactor.util.context.Context;",
+            "",
+            "class A {",
+            "  int[] negative1() {",
```

Comment:
Keeping these counters in sync negatively impacts maintenance. Maybe in a separate PR we should consider a more compact test format for these matchers. (But for this PR I followed the established approach.)